clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/ClioIndex.scala

```diff
@@ -35,7 +35,7 @@ trait ClioIndex {
 
   val queryInputTag: ClassTag[QueryInputType]
 
-  val keyEncoder: Encoder[KeyType]
+  val objectKeyEncoder: ObjectEncoder[KeyType]
```

Comment:
We could leave this named as it was, and just change the type (doesn't matter to me either way, just a less noisy diff)

Reply:
:+1: